content/blog/2025-04-04-tmil-15/index.md

```diff
@@ -0,0 +1,212 @@
++++
+title = "Linebender in March 2025"
+authors = ["Daniel McNab", "Bruce Mitchener"]
++++
+
+Linebender is an informal open-source organization working on various projects to advance the state of the art in GUI for [the Rust programming language](https://rust-lang.org).
+
+## Masonry
+
+Masonry is the widget system developed by Linebender.
+It provides a non-opinionated retained widget tree, designed as a base layer for high-level GUI frameworks.
+
+We have made a major change to the internal design of Masonry, in [xilem#910][] and [xilem#914][].
+This change splits Masonry into two packages, `masonry_core` and `masonry`.
+The Masonry crate provides a runner for Masonry core using Winit, and should be used for an integrated app development experience.
+Masonry Core contains the implementation of Masonry's widgets and passes, and is designed to be be embedded into existing apps (currently only those which use wgpu).
+This should be transparent to existing Masonry users.
+
+We hoped to release a new alpha of Xilem and Masonry to crates.io in March.
+However, this was put on hold whilst the Properties experiment we mentioned last month continues:
+
+- [xilem#873][]: Implements properties, which is additional data attached to each widget in a uniform way, intended to enable styling.
+- [xilem#892][]: Uses properties for styling the built-in `Button` widget.
+
+<!-- TODO: Screenshot of new capability from button styling. Deferred to Olivier. Non-blocking -->
+
+We have also made significant progress outside of these highlighted areas:
+
+- [xilem#897][]: Reduced the size of many of our screenshot tests, to better.
+- [xilem#904][]: Implemented a new algorithm for screenshot testing, which better reflects the aspects of the screenshots we wish to test.
+  It also implements automatic compression of blessed screenshots using the excellent [oxipng][] library.
+- [xilem#899][]: Added documentation for lots of items and methods which were missing documentation.
+- [xilem#882][]: Adds the core of a virtual scrolling widget, to be used with medium or large scroll areas.
+
+<figure>
+<img style="height: auto" src="virtual-scroll.png" alt="A screenshot of a desktop window, showing a list of numbers around 999900. Every number divisible by 3 has been replaced with the word 'Fizz', 'Buzz' for 5, and 'FizzBuzz' for numbers divisible by both. The 'FizzBuzz' which would be item 999900 is twice as large as the other items." width="458" height="509">
+<figcaption>
+
+[xilem#882][] added a virtual scrolling widget. This is the 'Infinite FizzBuzz' example, which also shows how the virtual scrolling can support items with arbitrary heights.
```

Comment:
This PR is linked twice with different messages. We may also want to explain more about what a virtual scrolling widget is.

Reply:
This is a caption explaining the figure, the other is part of the updates of what PRs were merged. This pattern of mentioning a PR twice is one we've used a few time

Certainly adding a one-sentence explainer will be good. I'm just waiting for Raph to release the lock, then I can action this.